src/SubtitleEdit.csproj

```diff
@@ -1355,6 +1355,9 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="Resources\Transparent Background.png" />
+    <Content Include="..\LICENSE.txt">
+      <Link>LICENSE.txt</Link>

```

Comment:
> Is the "Link" really needed?

Yeah, I think so. It was added by VS2015 automatically.
Maybe it's a new tag in VS2015